SoulsFormats/Util/Kernel32.cs

```diff
@@ -8,11 +8,88 @@ namespace SoulsFormats
     public static class Kernel32 {
         [DllImport("kernel32", SetLastError = true)]
         static extern IntPtr LoadLibraryW([MarshalAs(UnmanagedType.LPWStr)]string lpFileName);
-        
+
         [DllImport("kernel32.dll")]
```

Comment:
Should this still have the ".dll" ending, given that it was removed from the other instances of DllImport?

Reply:
Since there's no kernel32.dll in linux, this line is just fine.
Actually, we use ` public static class Libdl` in linux instead.

I'm not sure if netstandard could use the latest [`NativeLibrary.Load`](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.nativelibrary.load?view=net-8.0) and/or [LibraryImport](https://learn.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke), if so, we could simplify some code here.